lib/cc/engine/analyzers/php/main.rb

```diff
@@ -14,31 +11,31 @@ class Main < CC::Engine::Analyzers::Base
           PATTERNS = [
             "**/*.php",
           ].freeze
-          DEFAULT_MASS_THRESHOLD = 28
+          DEFAULT_MASS_THRESHOLD = 75
           DEFAULT_FILTERS = [
-            "(use ___)".freeze,
+            "(Stmt_Use ___)",
           ].freeze
           POINTS_PER_OVERAGE = 100_000
+          REQUEST_PATH = "/php"
+          COMMENT_MATCHER = Sexp::Matcher.parse("(_ (comments ___) ___)")
```

Comment:
Is is possible to add this to `DEFAULT_FILTERS` instead of deleting the nodes?

Reply:
Unfortunately, it's not. That's the first thing I tried! Adding this to `DEFAULT_FILTERS` causes the entire function to be omitted from analysis.

Reply:
Hmmm, what about `(comments ___)` as a filter? Would that match just the comment instead of the entire function?

Reply:
It doesn't, please see the test output below.

I discussed this with @wfleming, too, and he said that while this is not entirely desirable behavior, this is how we originally intended filtering to work.

```

Failures:

  1) CC::Engine::Analyzers::Php::Main#run comments ignores PHPDoc comments
     Failure/Error: expect(issues.length).to be > 0
       expected: > 0
            got:   0
     # ./spec/cc/engine/analyzers/php/main_spec.rb:232:in `block (4 levels) in <top (required)>'
     # ./spec/spec_helper.rb:27:in `block (4 levels) in <top (required)>'
     # ./spec/spec_helper.rb:26:in `chdir'
     # ./spec/spec_helper.rb:26:in `block (3 levels) in <top (required)>'
     # ./spec/spec_helper.rb:23:in `block (2 levels) in <top (required)>'

  2) CC::Engine::Analyzers::Php::Main#run comments ignores one-line comments
     Failure/Error: expect(issues.length).to be > 0
       expected: > 0
            got:   0
     # ./spec/cc/engine/analyzers/php/main_spec.rb:266:in `block (4 levels) in <top (required)>'
     # ./spec/spec_helper.rb:27:in `block (4 levels) in <top (required)>'
     # ./spec/spec_helper.rb:26:in `chdir'
     # ./spec/spec_helper.rb:26:in `block (3 levels) in <top (required)>'
     # ./spec/spec_helper.rb:23:in `block (2 levels) in <top (required)>'
```

Reply:
> while this is not entirely desirable behavior, this is how we originally intended filtering to work

Well,. it's how the *author* of the filtering intended it to work, clearly. I'm not sure it's how "we" as in the product team intended it to work. I honestly don't know if if was what we intended or it was miscommunicated requirements & not appropriately QAed.